### PR TITLE
PR Controllers

### PR DESCRIPTION
**Pull request by DarioAlessioR**
PR Controllers for activity Blog app - controllers.

This PR includes the controller's Users and Posts, with the method's index and show. 

It also includes the routes and views for the methods described in Users and Posts.

Thank you in advance for your review.

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  resources :users, only: [:index, :show]
+  resources :posts, only: [:index, :show]
```
Comment (youmari) [inline config/routes.rb]: - [ ] Nice adding routes for your app however what you did it's not as required right now it create sperate routes for both of them but the routes should be as in the picture in the requirements so please try  to modify your code in order to make the same routes as in the requirements
- Required routes 
image
- Yours 
Screen Shot 2022-10-03 at 18 38 53

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @DarioAlessioR 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Great commit messages
✔️ PRO PR
✔️ No linter error 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/routes.rb (hunk 1 above)
Comment (DarioAlessioR) [reply config/routes.rb]: Hi @youmari .

When I run rails routes, I get exactly the required routes, as you can see in my terminal's screenshot:
image

So, I already have what you are asking for. Or I don't understand what you mean. Please, can you explain it again or tell me what exactly should I code as routes ?

By the way, I have reviewed 4 different repos approved last week with the same code as I did. So, I hope you understand why I'm confused by your required changes.

Thank you.

Comment (DarioAlessioR) [reply config/routes.rb]: Ok. Required changes done.
image

**Review by alexander16108 — APPROVED** [review]
Hi @DarioAlessioR , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 




## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/routes.rb (hunk 1 above)
Comment (youmari) [reply config/routes.rb]: @DarioAlessioR That's what I mean in order to get the required routes you've should nest the posts routes inside the user routes in order to get this result but at the first you wasn't doing that

**Pull request merged**